libcontainer/process_linux.go

```diff
@@ -76,6 +79,10 @@ func newProcessComm() (*processComm, error) {
 	if err != nil {
 		return nil, fmt.Errorf("unable to create init pipe: %w", err)
 	}
+	comm.stage1SockParent, comm.stage1SockChild, err = utils.NewSockPair("stage1")
+	if err != nil {
+		return nil, fmt.Errorf("unable to create init pipe: %w", err)
```

Comment:
Although it's stage-1, do we need another name instead of init pipe?

Reply:
Yes, thanks.

Reply:
fixed! Thanks. Welcome more reviews!